code/__DEFINES/mobs.dm

```diff
@@ -797,19 +797,19 @@ GLOBAL_LIST_INIT(human_body_parts, list(BODY_ZONE_HEAD,
 #define IGNORE_HAND (1<<1)
 
 #define TIER_ONE_YOUNG_THRESHOLD 60
-#define TIER_ONE_MATURE_THRESHOLD 120
-#define TIER_ONE_ELDER_THRESHOLD 240
-#define TIER_ONE_ANCIENT_THRESHOLD 240
+#define TIER_ONE_MATURE_THRESHOLD 180
+#define TIER_ONE_ELDER_THRESHOLD 420
+#define TIER_ONE_ANCIENT_THRESHOLD 660
 
 #define TIER_TWO_YOUNG_THRESHOLD 120
-#define TIER_TWO_MATURE_THRESHOLD 240
-#define TIER_TWO_ELDER_THRESHOLD 480
-#define TIER_TWO_ANCIENT_THRESHOLD 240
+#define TIER_TWO_MATURE_THRESHOLD 360
+#define TIER_TWO_ELDER_THRESHOLD 840
+#define TIER_TWO_ANCIENT_THRESHOLD 1080
 
 #define TIER_THREE_YOUNG_THRESHOLD 250
-#define TIER_THREE_MATURE_THRESHOLD 500
-#define TIER_THREE_ELDER_THRESHOLD 1000
-#define TIER_THREE_ANCIENT_THRESHOLD 100
+#define TIER_THREE_MATURE_THRESHOLD 750
+#define TIER_THREE_ELDER_THRESHOLD 1750
+#define TIER_THREE_ANCIENT_THRESHOLD 1850
```

Comment:
Why nto change the macro to TIER_ONE_MATURE_THRESHOLD 120+TIER_ONE_YOUNG_THRESHOLD